transform/mattermost-analytics/models/marts/marketing/fct_marketing_campaign_member.sql

```diff
@@ -0,0 +1,38 @@
+with marketing as (
+    select marketing_id
+        , email as email
+        , server_id
+        , created_at
+        , updated_at
+    from
+        {{ ref('stg_cws__marketing')}}
+    where subscribed_content = 'security_newsletter'
+    qualify row_number() over (
+            partition by email
+            order by
+                created_at desc
+        ) = 1
+),
+marketing_campaign_member as (
```

Comment:
Same obs here - we talk about specific campaign members ( security_newsletter related), not **all** the marketing campaign members.